xds/src/test/java/io/grpc/xds/internal/sds/SdsProtocolNegotiatorsTest.java

```diff
@@ -239,10 +244,14 @@ public SocketAddress localAddress() {
   }
 
   @Test
-  public void serverSdsHandler_nullTlsContext_expectPlaintext() throws IOException {
+  public void serverSdsHandler_nullTlsContext_expectFallbackProtocolNegotiator() {
+    ChannelHandler mockChannelHandler = mock(ChannelHandler.class);
+    ProtocolNegotiator mockProtocolNegotiator = mock(ProtocolNegotiator.class);
+    when(mockProtocolNegotiator.newHandler(grpcHandler)).thenReturn(mockChannelHandler);
     SdsProtocolNegotiators.HandlerPickerHandler handlerPickerHandler =
         new SdsProtocolNegotiators.HandlerPickerHandler(
-            grpcHandler, /* xdsClientWrapperForServerSds= */ null);
+            grpcHandler, /* xdsClientWrapperForServerSds= */ null,
+            mockProtocolNegotiator);  // new FallbackProtocolNegotiator()
```

Comment:
nit: this comment (ed code?) is confusing

Reply:
oops, good catch. Will fix